03_Modules/Configuration/test/providers/bootConfig.ts

```diff
@@ -0,0 +1,77 @@
+import { Boot } from '@citrineos/data';
+import {
+  AttributeEnumType,
+  RegistrationStatusEnumType,
+  type SetVariableResultType,
+  SetVariableStatusEnumType,
+} from '@citrineos/base';
+import { faker } from '@faker-js/faker';
+
+type UpdateFunction<T> = (item: T) => void;
+
+export const aValidSetVariableResult = (
+  updateFunction?: UpdateFunction<SetVariableResultType>,
+): SetVariableResultType => {
+  const item: SetVariableResultType = {
+    attributeStatus: SetVariableStatusEnumType.Accepted,
+    component: {
+      name: faker.lorem.word(),
+      instance: faker.string.uuid(),
+    },
+    variable: {
+      name: faker.lorem.word(),
+      instance: faker.string.uuid(),
+    },
+    customData: {
+      vendorId: faker.string.uuid(),
+    },
+    attributeType: AttributeEnumType.Actual,
+    attributeStatusInfo: {
+      reasonCode: faker.lorem.word(),
+      additionalInfo: faker.lorem.sentence(),
+    },
+  };
+
+  if (updateFunction) {
+    updateFunction(item);
+  }
+
+  return item;
+};
+
+export const aValidBootConfig = (
+  updateFunction?: UpdateFunction<Boot>,
+): Boot => {
+  const item: Boot = {
+    id: faker.string.uuid(),
+    lastBootTime: faker.date.recent().toISOString(),
+    heartbeatInterval: faker.number.int({ min: 30, max: 3600 }),
+    bootRetryInterval: faker.number.int({ min: 30, max: 3600 }),
+    status: RegistrationStatusEnumType.Pending,
+    getBaseReportOnPending: faker.datatype.boolean(),
+    variablesRejectedOnLastBoot: [aValidSetVariableResult()],
+    bootWithRejectedVariables: faker.datatype.boolean(),
+  } as Boot;
```

Comment:
This `as Boot` should not be needed here.

Reply:
I think it's needed, else there are issues with it being a class instance instead of an object. And it can't be a class instance here as we want to avoid having to deal with the squelize part of it. 
With out the `as Boot` it freaks out that it's missing the sequelize added things such as `$get`